Sources/FaunaDB/QueryResult.swift

```diff
@@ -98,6 +196,13 @@ extension QueryResult {
 
 extension QueryResult {
 
+    /**
+        Execute a callback when the resulting value is available.
```

Comment:
this is a LITTLE weird verbiage when describing a future, perhaps: "Executes the specified callback when the QueryResult is successful"? Not 100% sure on that so ill think